core/tests/puppeteer-acceptance-tests/user-utilities/exploration-editor-utils.ts

```diff
@@ -872,6 +876,92 @@ export class ExplorationEditor extends BaseUser {
     }
     await this.clickOn(previewRestartButton);
   }
+
+  /**
+   * Function for creating an exploration with only EndExploration interaction with given title.
+   */
+  async createExplorationWithOnlyEndInteraction(
+    title: string
+  ): Promise<string | null> {
```

Comment:
Is there any specific reason to make a new method here?
Please refer previous comment.


You can easily create an exploration like this ->
```typescript
      await explorationEditor.navigateToCreatorDashboardPage();
      await explorationEditor.navigateToExplorationEditorPage();
      await explorationEditor.dismissWelcomeModal();

      await explorationEditor.createExplorationWithMinimumContent(
        'Exploration one',
        INTERACTION_TYPES.END_EXPLORATION
      );
      await explorationEditor.saveExplorationDraft();
      explorationId = await explorationEditor.publishExplorationWithContent(
        'Exploration one',
        'Exploration one',
        'Algebra'
      );
      
```